src/components/language/language.service.ts

```diff
@@ -373,7 +380,7 @@ export class LanguageService {
       },
       sensitivity: props.sensitivity,
       ethnologue,
-      canDelete: true, // TODO
+      canDelete: await this.db.checkDeletePermission(langId, session),
```

Comment:
Tried to move this into its own resolver to save on performance, but got an error in this return statement because the Resource dto needs a `canDelete` property.  Not sure how to get around this or if it's a good idea.